src/addAndRemove.js

```diff
@@ -0,0 +1,21 @@
+export const deleteTask = (id, tasksArray) => {
+  const newFilteredTask = tasksArray.filter((task) => task.id !== id);
+  const deleteTask = document.getElementById(`${id}`);
+  deleteTask.remove();
+  tasksArray = newFilteredTask;
+  localStorage.setItem('tasksArray', JSON.stringify(tasksArray));
+};
```

Comment:
- [ ] Every time a task is removed, please update the indexes in the remaining elements from the tasks array. The indexes are always supposed to be consecutive numbers, beginning from the first element in the array and finishing in the last one.

Reply:
This has been done automatically using the `array.filter` method, the index of the array has been rearranged and reassigned to the new elements in the new array, `newFilteredTask`